### PR TITLE
MFW-4523 Added a mteric field to route schema

### PR DESCRIPTION
**Pull request by vishalmane09**
Problem: Add 'metric' property in static routes schema

Changes: added a new field 'metric' to route schema

Testing instructions:
Run the test cases by using command ./validate.py in mfw_schema
Check all test cases passed or not

Expected Result: 
All the test cases from route_schema should pass
Please refer below output for reference
```
=== START OF MODULE TEST: <module 'v1.routes.validate_routes' from '/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/validate_routes.py'> ===

WARNING: Using default json filename:/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/test_routes.json
WARNING: Using default schema filename:/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/test_schema.json
test_routes (v1.routes.validate_routes.TestRoutesSchema)
validates routes ... ok

----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
=== END OF MODULE TEST: <module 'v1.routes.validate_routes' from '/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/validate_routes.py'> ===

```

### CODE REVIEW

File: v1/routes/validate_routes.py [hunk 1]
```diff
@@ -34,5 +34,10 @@ def test_routes(self):
         routes = self.json_data["routes"]
         self.assertEqual(len(routes), 2,  "Invalid routes content")
 
+        for i, route in enumerate(routes):
+            metric_value = route.get("metric", None)
+            if metric_value is not None:
```
Comment (yogesh-chhetri) [inline v1/routes/validate_routes.py:39]: `if metric_value is not None:`
can simply be written as
`if metric_value:`
*makes code mode readable

Comment (vishalmane09) [reply v1/routes/validate_routes.py]: Done

**Review by yogesh-chhetri — APPROVED** [review]
LGTM

File: v1/routes/test_routes.json [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
+        "metric": 0
       },
       {
         "enabled": true,
         "description": "route 2",
         "interfaceId": 1,
         "network": "wlan",
-        "nextHop": "192.168.1.2"
+        "nextHop": "192.168.1.2",
+         "metric": 0
```
Comment (manuwelakanade) [inline v1/routes/test_routes.json:17]: indentation issue

Comment (vishalmane09) [reply v1/routes/test_routes.json]: Fixed indentation issue

**Pull request closed without merging**